packages/odie-client/src/data/index.ts

```diff
@@ -28,6 +28,8 @@ export const useGetOdieStorage = ( key: OdieStorageKey ) => {
 				path: `/help-center/odie/history/last-chat-id`,
 			} as APIFetchOptions ).then( ( res ) => res[ storageKey ] );
 		},
+		// Give this further thought.
+		staleTime: 500, // 0.5 minute,
```

Comment:
Might be missing something, but shouldn't this be in seconds? 

```suggestion
		staleTime: 500, // 0.5 seconds
```

Reply:
Great catch. Thanks!